zboss/include/osif/zb_revision_r22.h

```diff
@@ -6,5 +6,5 @@
  * This file implements mappings of Kconfig options enabling and configuring ZBOSS features.
  */
 
-/* ZBOSS platform for NCS build number. */
+/* ZBOSS platform for NCS revision number. */

```

Comment:
`ZBOSS platform for NCS, revision number`
is more readable for me

Reply:
I prefer: `Revision number of ZBOSS platform for NCS.`. What do you think? :slightly_smiling_face:

Reply:
very readable :)